rc/lsp.kak

```diff
@@ -999,10 +1138,17 @@ define-command -hidden lsp-show-hover -params 3 -docstring %{
 
     content=$(printf %s "$content" | sed s/\'/\'\'/g)
 
-    case $kak_opt_lsp_hover_anchor in
-        true) printf "info -markup -anchor %%arg{1} -- '%s'" "$content";;
-        *)    printf "info -markup -- '%s'" "$content";;
+    case $1 in
+        modal) printf "info -markup -style modal -- '%s'" "$content";;
+        above) printf "info -markup -style above -- '%s'" "$content";;
```

Comment:
only the "modal" style is currently used, so I'd remove the other cases.
(also "below" doesn't seem to work).

If we allow them we'd write it as
`above|below|menu|modal) printf "info -markup -style $1 -- '%s'" "$content";;`

Reply:
Removed other cases